src/admin-views/editor/panel/ticket.php

```diff
@@ -195,7 +195,7 @@ class="ticket_field ticket_provider"
 				/** @var Tribe__Tickets__Admin__Views $admin_views */
 				$admin_views = tribe( 'tickets.admin.views' );
 
-				$admin_views->template( 'editor/fieldset/advanced', [ 'post_id' => $post_id, 'ticket_id' => $ticket_id ] );
+				$admin_views->template( 'editor/fieldset/advanced', [ 'post_id' => $post_id, 'ticket_id' => $ticket_id, 'provider' => $provider ] );
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: f11828d14b2f826a0f7517af50971fae</sub>

Reply:
```suggestion
				$admin_context = [
					'post_id'   => $post_id,
					'ticket_id' => $ticket_id,
					'provider'  => $provider,
				];

				$admin_views->template( 'editor/fieldset/advanced', $admin_context );
```